src/lib/calculate-polygon-center.js

```diff
@@ -0,0 +1,165 @@
+import { Client } from "pg";
+const client = new Client();
+client.connect();
+
+// client.query(`CREATE TABLE Polygons (
+// 	type Text,
+// 	properties_name Text,
+// 	properties_type Text,
+// 	properties_pop2001 Numeric,
+// 	properties_dwell2001 Numeric,
+// 	properties_areaSqkm Numeric,
+// 	properties_popSqkm Numeric,
+// 	properties_province Text,
+// 	properties_centerLat Numeric,
+// 	properties_centerLng Numeric,
+// 	properties_lowLat Numeric,
+// 	properties_lowLng Numeric,
+// 	properties_highLat Numeric,
+// 	properties_highLng Numeric,
+// 	properties_radius Numeric,
+// 	geometry JSON
+// );`);
+
+client.query("SELECT * FROM myTable", (err, res) => {
+  const formattedPolygons = res.rows.map(row => {
+    const {
+      centerLat,
+      centerLng,
+      lowLat,
+      lowLng,
+      highLat,
+      highLng,
+      radius
+    } = getPolygonLimits({
+      coordinates: JSON.parse(row.geometrycoordinates),
+      geometryType: row.geometrytype
+    });
+
+    return {
+      type: row.type,
+      properties: {
+        name: row.propertiesname,
+        type: row.propertiestype,
+        pop2001: row.propertiespop2001,
+        dwell2001: row.propertiesdwell2001,
+        areaSqkm: row.propertiesarea_sqkm,
+        popSqkm: row.propertiespop_sqkm,
+        province: row.propertiesprov,
+        centerLat,
+        centerLng,
+        lowLat,
+        lowLng,
+        highLat,
+        highLng,
+        radius
+      },
+      geometry: {
+        type: row.geometrytype,
+        coordinates: JSON.parse(row.geometrycoordinates)
+      }
+    };
+  });
+  const insertPromises = formattedPolygons.map(geojson => {
+    return saveGeoJsonToDatabase({ geojson });
+  });
+
+  Promise.all(insertPromises)
```

Comment:
We can use async/await here :)

Reply:
Okay I will need to do a tutorial on that. Never used it before, but I'm all about learning best practices 🤓